charts/budibase/templates/app-service-deployment.yaml

```diff
@@ -80,6 +80,10 @@ spec:
           value: {{ .Values.services.objectStore.url }}
         - name: PLUGIN_BUCKET_NAME
           value: {{ .Values.services.objectStore.pluginBucketName | default "plugins" | quote }}
+        - name: APPS_BUCKET_NAME
+          value: {{ .Values.services.objectStore.appsBucketName | default "apps" | quote }}
+        - name: GLOBAL_CLOUD_BUCKET_NAME
```

Comment:
Should we just call it `GLOBAL_BUCKET_NAME`? Given multi tenancy is in our future for self hosters

Reply:
Probably - I just mapped this from the name it currently has in `server` for consistency.